pkg/generate/code/check.go

```diff
@@ -45,6 +46,10 @@ func CheckExceptionMessagePrefix(
 			return fmt.Sprintf("&& strings.HasPrefix(awsErr.Message(), \"%s\") ",
 				*excConfig.MessagePrefix)
 		}
+		if ok && excConfig.MessageSuffix != nil {
```

Comment:
nit: Replace the `if ok` in each of these conditional blocks with an early return at the top:
```go
if !ok {
  return ""
}
```

Reply:
replaced